### ISSUE TITLE
qvestion

### ISSUE THREAD

**Issue by SniexDejjtik** [necr0n/dcs-make-flyable#16]
Do you gonna upd the Mods when ED gonna fix the stuff that they make bcs rgn the AI planes some of them are not flyable at all.
so im yust asking if u gonna upd this when everything gonna be fixed, or i would need to change stuff at my own later.

**Comment by necr0n** [comment]
Yes, I'll be happy to update the mods when ED fixes the models. Feel free to submit merge request as well.
